src/completion.ts

```diff
@@ -50,28 +50,30 @@ export function asCompletionItem(entry: tsp.CompletionEntry, optionalReplacement
         item.sortText = '\uffff' + entry.sortText;
     }
 
-    const { isSnippet, sourceDisplay } = entry;
+    const { isSnippet, replacementSpan, sourceDisplay } = entry;
     if (isSnippet && !features.completionSnippets) {
         return null;
     }
-    if (features.completionSnippets && (isSnippet || entry.isImportStatementCompletion || item.kind === lsp.CompletionItemKind.Function || item.kind === lsp.CompletionItemKind.Method)) {
+    if (features.completionSnippets && (isSnippet || item.kind === lsp.CompletionItemKind.Function || item.kind === lsp.CompletionItemKind.Method)) {
```

Comment:
The `entry.isImportStatementCompletion` check was removed it as it's only provided for telemetry reasons and `isSnippet` check covers that case already.